pyQuARC/version.py

```diff
@@ -0,0 +1 @@
+__version__="1.1.0"
```

Comment:
this could be a broader discussion within the team, but we need some automated linting.